README.md

```diff
@@ -33,13 +33,21 @@ appveyor_agent '3.12.0' do
 end
 ```
 
-### Deploy
-`appveyor_deploy` start the deployment for the specified environment in AppVeyor
+### Start Deployment
+The `appveyorapi_deploy` LWRP can be used to start the deployment for the specified environment in Appveyor CI using its API.
+
 ```ruby
```

Comment:
From Infra perspective, I felt environment name is vital than the build version.